index.html

```diff
@@ -65,6 +65,73 @@ <h2 class="i-m">
             </li>
         </ul>
     </main>
+    <section class="works">
+        <h1 class="head-works">My Recent Works</h1>
+        <hr>
+        <section class="content">
```

Comment:
- [ ] Your works section is required to have multiple cards displaying different works as shown in the image below but you currently have only one card. please go over to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=48-27&t=E1OnAzxwAvgq2Dya-0) and add these designs for each card to your over all project. it is important that this project is presented closely as shown on the figma template as this shows tour attention to details as a developer and it is a good quality recruiters search for :+1: :sparkles: 
- figma design
![mul](https://github.com/AndresEspi/portafolio/assets/89055502/f83a39d9-ccdf-48a6-a637-d281979a3280)